mac-nfs.readme.md

```diff
@@ -1,33 +0,0 @@
-
-# Setup NFS (for mac/OSX users)
-
-## First-time setup of NFS
-
-If you want to use NFS, you'll need to do various things the very first time.
-
-`/etc/exports` should contain information about your code folder
-
-Assuming you're keeping your Docker/code projects in `~/code`, you'll want to add
-this line to your `/etc/exports`:
-
-`/System/Volumes/Data/Users/<USERNAME>/code -mapall=<USERNAME>:staff -alldirs localhost`
-
-## Full access
-
-`/sbin/nfsd` and your terminal program needs "Full disk access":
-
-![OSX preference pane providing access to NFSD](documentation/images/nftd.png)
```

Comment:
Please remember to delete the image at `documentation/images/nftd.png`.

Reply:
Tes 👍 Done